http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -69,7 +68,7 @@
 
     private static final String CLIENT_NAME = "NettyNio";
 
-    private static final Logger log = LoggerFactory.getLogger(NettyNioAsyncHttpClient.class);
+    private static final NettyClientLogger log = NettyClientLogger.getLogger(NettyNioAsyncHttpClient.class);
```

Comment:
Could you please help me to confirm if there will be no changes in the way the User can enable Debug logs as mentioned in https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/logging-slf4j.html ?
apart from adding 
`log4j.logger.io.netty.handler.logging.LoggingHandler=DEBUG`
If there is a difference in how user can enable the logs now ,can we document this in this PR  ?

Reply:
No it doesn't change those details. The logger name is based on the class name given to the `getLogger` method, which stays the same.